app/helpers/application_helper.rb

```diff
@@ -34,4 +34,8 @@ def access_denied
     redirect_back(fallback_location: root_path)
     flash[:alert] = 'Доступ заборонено'
   end
+
+  def first_attached_image(issue)
```

Comment:
guys ... I want to see 10x smaller amount of such methods
first try to implement logic in model
then if model is not good place in helper